docs/src/writing-tests-csharp.md

```diff
@@ -71,7 +71,7 @@ public class UnitTest1 : PageTest
         await Expect(Page).ToHaveTitleAsync(new Regex("Playwright"));
 
         // create a locator
-        var getStarted = Page.Locator("text=Get Started");
+        var getStarted = Page.GetByRole(AriaRole.Link, new () { Name = "Get started" });
```

Comment:
style nit: I saw Max fixing options code to be formatted as `new() {` instead of `new () {`